MapboxDirections/MBDirections.swift

```diff
@@ -36,10 +37,7 @@ public class MBDirections: NSObject {
 
     private let request: MBDirectionsRequest
     private let configuration: MBDirectionsConfiguration
-    private var task: NSURLSessionDataTask?
-    public var calculating: Bool {
```

Comment:
Per voice, given the async'ness of requests this bit of compatibility with MapKit isn't worthwhile: our directions SDK doesn't have any good reason will permit concurrent requests and any kind of serialization would happen on the app level.